spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -252,6 +252,23 @@ class CodeEditor(TextEditBaseWidget):
     #: Signal emitted when a key is pressed
     key_pressed = Signal(QKeyEvent)
 
+    #: Signal emitted when a key is released
+    key_released = Signal(QKeyEvent)

```

Comment:
Please prefix new signals with `sig_`, i.e. `sig_key_released`.

We should have done that when we move from PyQt4 to 5, but unfortunately we didn't do it.